orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/EvaluateVariablesStage.java

```diff
@@ -35,6 +39,36 @@ public void taskGraph(@Nonnull Stage stage, @Nonnull TaskNode.Builder builder) {
     builder.withTask("evaluateVariables", EvaluateVariablesTask.class);
   }
 
+  @Override
+  public void processExpressions(
+      @Nonnull Stage stage,
+      @Nonnull ContextParameterProcessor contextParameterProcessor,
+      @Nonnull ExpressionEvaluationSummary summary) {
+
+    EvaluateVariablesStageContext context = stage.mapTo(EvaluateVariablesStageContext.class);
+    StageContext augmentedContext = contextParameterProcessor.buildExecutionContext(stage);
+    Map<String, Object> varSourceToEval = new HashMap<>();
+    List<Variable> variables =
+        Optional.ofNullable(context.getVariables()).orElse(Collections.emptyList());
+
+    for (Variable var : variables) {
+      if (var.getValue() instanceof String) {
+
+        var.saveSourceValue();
+        varSourceToEval.put("var", var.getValue());
+
+        Map<String, Object> evaluatedVar =
+            contextParameterProcessor.process(varSourceToEval, augmentedContext, false, summary);
+
+        var.setValue(evaluatedVar.get("var"));
+        augmentedContext.put(var.key, var.value);
+      }
+    }
+
```

Comment:
Question ? Don't we have a ObjectMapper wired up to use as a Spring bean ?  May be we can inject that and use that instead of newing up ObjectMapper here.

Reply:
+1

Reply:
yup, good point, thanks!